prime-router/docs/api/delivery.yml

```diff
@@ -0,0 +1,286 @@
+openapi: 3.0.2
+info:
+  title: Prime ReportStream
+  description: A router of public health data from multiple senders and receivers
+  contact:
+    name: USDS at Centers for Disease Control and Prevention
+    url: https://reportstream.cdc.gov
+    email: reportstream@cdc.gov
+  version: 0.2.0-oas3
+paths:
+  /v1/receivers/{receiverFullName}/deliveries:
```

Comment:
POST parameters do not occur in the path.

Also, we should have a consensus on how to handle API versioning going forward.

Reply:
I'll try to pull together a doc for engineering sync next week on API versioning, but it's a pattern we've started adopting on the platform team in order to keep APIs backwards compatible.

WRT to the path params, I had incorrectly labeled them as being in the `query` rather than `path` and have just updated them.  Let me know if this isn't what you were referring to.